behat/tests/bootstrap/v2/reporting/sections/AccountsSectionTrait.php

```diff
@@ -0,0 +1,490 @@
+          'row' => $isNdr ? 'div.govuk-summary-list__row' : 'tr',
+          'head' => $isNdr ? 'dt' : 'th',
+          'data' => $isNdr ? 'dd' : 'td'
+        ];
+
+        $this->iAmOnAccountsSummaryPage();
+
+        $tableBody = $this->getSession()->getPage()->find('css', $accountSummaryElems['tableBody']);
+
+        if (!$tableBody) {
+            $this->throwContextualException('A tbody element was not found on the page');
+        }
+
+        $tableRows = $tableBody->findAll('css', $accountSummaryElems['row']);
+
+        if (!$tableRows) {
+            $this->throwContextualException('A tr element was not found on the page');
+        }
+
+        if ($isNdr) {
+            unset($tableRows[0]);
+            $tableRows = array_values($tableRows);
+        }
+
+        foreach ($tableRows as $tRowKey=>$tableRow) {
+            $tableHeader = $tableRow->find('css', $accountSummaryElems['head']);
+            $headHtml = trim(strtolower($tableHeader->getHtml()));
+            assert(
+                str_contains($headHtml, $this->accountList[$tRowKey]['accountType']),
+                sprintf('matching account %s ', $this->accountList[$tRowKey]['accountType'])
+            );
+            assert(
+                str_contains($headHtml, $this->accountList[$tRowKey]['name']),
+                sprintf('matching name %s ', $this->accountList[$tRowKey]['name'])
+            );
+            assert(
+                str_contains($headHtml, $this->accountList[$tRowKey]['accountNumber']),
+                sprintf('matching account number %s ', $this->accountList[$tRowKey]['accountNumber'])
+            );
+            $sortCode = str_replace('-', '', $this->accountList[$tRowKey]['sortCode']);
+            assert(
+                str_contains($headHtml, $sortCode),
+                sprintf('matching sort code %s ', $sortCode)
+            );
+            assert(
+                str_contains($headHtml, $this->accountList[$tRowKey]['joint']),
+                sprintf('matching sort code %s ', $this->accountList[$tRowKey]['joint'])
+            );
+
+            $tableFields = $tableRow->findAll('css', $accountSummaryElems['data']);
+
+            foreach ($tableFields as $tFieldKey=>$tableField) {
+                $balanceItem = trim(strtolower($tableField->getHtml()));
+                if ($tFieldKey == 0) {
+                    assert(
+                        str_contains($balanceItem, $this->accountList[$tRowKey]['openingBalance']),
+                        $this->accountList[$tRowKey]['openingBalance']
+                    );
+                } elseif ($tFieldKey == 1 and $this->reportUrlPrefix != 'ndr') {
+                    assert(
+                        str_contains($balanceItem, $this->accountList[$tRowKey]['closingBalance']),
+                        $this->accountList[$tRowKey]['closingBalance']
+                    );
+                }
+            }
+        }
+    }
+
+    /**
+     * @When I add a couple of new accounts
```

Comment:
I like couple because it does matter that it's two because removal of takes you to different place than removal of second. It's also only in this features trait so doesn't need to be super flexible